schemas/access/permission-1.yml

```diff
@@ -105,10 +162,16 @@ oneOf:
       - github-org-team
     org:
       type: string
+      description: |
+        The GitHub organization associated with the team.
```

Comment:
can you check if `description` content need to be duplicated inside `oneOf`, or only define it once in root level.